sysutils/SilentKnight/Portfile

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                SilentKnight
+version             1.6
+categories          sysutils aqua
+platforms           darwin
+supported_archs     x86_64
+license             freeware
```

Comment:
"freeware" is not one of the [recognized license names](https://github.com/macports/macports-infrastructure/blob/master/jobs/distributable_lib.tcl).

Reply:
Good catch, thanks. I've grepped the ports tree for existing license names and found that one, matching exactly what the original author uses on hit website. Now I see it's just a single case (sysutils/isightcapture), `| sort -u` hid that from me.

Will change it to "Permissive".

Reply:
> "freeware" is not one of the [recognized license names](https://github.com/macports/macports-infrastructure/blob/master/jobs/distributable_lib.tcl).

Btw. wouldn't it make sense to teach `port lint` to protest if a non-recognized license is used?

Reply:
> Now I see it's just a single case (sysutils/isightcapture)

And oops it looks like I'm the one who added that to that port 10 years ago. I'll fix that. That port incidentally is an existing example of a port that installs a binary because source was not available.

Reply:
> wouldn't it make sense to teach `port lint` to protest if a non-recognized license is used?

That's a reasonable idea but at present MacPorts base does not contain a list of known licenses. Only the port_binary_distributable.tcl script does (and I think it only contains a list of known distributable licenses; any other license (whether it's known or not) is considered not distributable). It has been suggested before that we should move that functionality into MacPorts base.

Reply:
I filed https://trac.macports.org/ticket/60315 and https://trac.macports.org/ticket/60316 for this.